src/app/ui-library/snackBar.tsx

```diff
@@ -0,0 +1,72 @@
+import toast from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+type ToastType = 'success' | 'error' | 'neutral';
+
+interface ToastProps {
+  text: string;
+  type: ToastType;
+}
+
+const getBackgroundColor = (type: ToastType, theme: any): string => {
+  const colors = {
+    success: theme.colors.feedback.success,
+    error: theme.colors.feedback.error,
+    neutral: theme.colors.feedback.neutral,
+  };
+  return colors[type] || theme.colors.feedback.neutral;
+};
+
+const getTextColor = (type: ToastType, theme: any): string => {
+  const colors = {
+    success: theme.colors.elevation0,
+    error: theme.colors.white_0,
+    neutral: theme.colors.white_0,
+  };
+  return colors[type] || theme.colors.elevation0;
+};
+
+const ToastContainer = styled.div<{ type: ToastType }>`
+  display: flex;
+  flex-direction: row;
+  background: ${(props) => getBackgroundColor(props.type, props.theme)};
+  border-radius: 12px;
+  box-shadow: 0px 7px 16px -4px rgba(25, 25, 48, 0.25);
+  height: 44px;
+  padding: 12px 20px;
+  width: 306px;
```

Comment:
thanks for making the ui-library component :)!

minor point - snackbar should shrink to hug the content unless on two lines, according to zeroheight:
https://zeroheight.com/0683c9fa7/p/562fac-snackbar/b/90234e

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/21f50ee4-41a0-47bb-823b-f84fbb38fb9e)